conf/systemd/packetfence-snmptrapd.service

```diff
@@ -2,7 +2,7 @@
 [Unit]
 Description=PacketFence Simple Network Management Protocol (SNMP) Trap Daemon.
 After=packetfence-base.target
-Wants=packetfence-base.target packetfence-config.service packetfence-iptables.service
+Wants=packetfence-base.target packetfence-config.service packetfence-firewalld.service
 
```

Comment:
you have to allow traffic through the firewall

Reply:
Hum, not able to find ports, will check on monday

Reply:
Done